### ISSUE TITLE
Accents show one day earlier

### ISSUE THREAD

**Issue by z29591259** [recruit-mp/LightCalendarView#18]
I call setAccents in "onMonthSelected" function
but accents always show one day earlier, e.g. 2017-01-19 shown on 2017-01-18
the date in Date object is definitely correct, because I had checked it in Log.

This is a nice tool, I hope this issue could be solved. Thanks ^ ^

**Comment by lpfmailtest** [comment]
Yes, you are right. I am also getting same issue. If i set 6th then Calender showing me a dot in 5th.

**Comment by TKolbusz** [comment]
I guess its the same issue I experienced when selecting the date, daysAfter func in DayLayout returns wrong number

**Comment by z29591259** [comment]
@TKolbusz I tested onDateSelected func, the Date object is correct.

**Comment by TKolbusz** [comment]
@z29591259 It is not about date object, it is fine. daysAfter function returns its result based on  hours, minutes and seconds, not only days, thats why it doesn't work as expected

**Comment by capitancooker** [comment]
Seems to be a time zone issue.
I try to add 22 hours and accents is on right date. All my dates are GMT+1.
Is a hugly workaround, but work. Hope is fixed soon.

**Comment by csbenz** [comment]
Getting the same issue. I suppose the lib is using a Japanese Zimezone ? :) 🗾
